src/frontend/components/usa--header.tsx

```diff
@@ -20,25 +21,40 @@ export function USAHeader() {
   ];
 
   return(
-    <Header basic={true} showMobileOverlay={expanded}>
-       <div className="usa-nav-container padding-top-1">
-          <div className="usa-navbar">
-            <Title>
-              <Image
-                src={PGovLogo}
-                width={56}
-                height={56}
-                alt={"Performance.gov logo"}
-                priority
-                className=""
-              />
-            </Title>
-            <NavMenuButton onClick={onClick} label="Menu" />
-          </div>
-          <PrimaryNav items={menuItems} mobileExpanded={expanded} onToggleMobileNav={onClick}>
-            <Search size="small" onSubmit={(e) => {e.preventDefault(); console.log("Search");}} />
-          </PrimaryNav>
-        </div>
-    </Header>
+    <header className="site-header dark-blue-bg grid-row padding-x-3">
+      <Image
```

Comment:
gotta add an aria-label to this component or we get a warning about this approximately 500 times during the build: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role#associated_wai-aria_roles_states_and_properties